chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/xmlgenerator/CertifiedProductSearchDetailsXmlGenerator.java

```diff
@@ -36,7 +36,9 @@ public static void add(CertifiedProductSearchDetails cp, String rootNodeName, XM
                 sw.writeStartElement("certificationEdition");
                 for (Entry<String, Object> entry : cp.getCertificationEdition().entrySet()) {
                     sw.writeStartElement("entry");
-                    createSimpleElement(entry.getKey(), "key", sw);
+                    if (entry.getKey() != null) {
+                        createSimpleElement(entry.getKey(), "key", sw);
+                    }
                     createSimpleElement(entry.getValue().toString(), "value", sw);
```

Comment:
For all of the other ones you also do a nullcheck on the value. Is there a reason not to do it here?

Reply:
Cut and paste error.  I'll fix.